### PR TITLE
Productinfo

### PR DESCRIPTION
**Pull request by leonardodeutsch**
hey @ks10825n @kyelindholm 

-Added product information
-Implemented componentDidUpdate in Overview.jsx
-Commented out MomentLocalesPlugins() in webpack.config.js because it was not defined, even after npm installing moment. @ks10825n let me know how to get it to work, I ran npm install and npm run react-dev would not run because line 45 of webpack.config.js has an error.
-Removed the conditional rendering of the Overview component in App.jsx

### CODE REVIEW

File: client/src/components/Overview/Overview.jsx [hunk 1]
```diff
@@ -33,6 +43,20 @@ class Overview extends React.Component {
           styles: response.data.results,
           currentPhoto: response.data.results[0].photos[0].url,
           selectedStylePhotos: response.data.results[0].photos,
+          price: response.data.results[0].original_price
+        })
+      })
+      .catch((error) => {
+        console.log(error); // do something with error or throw error
```
Comment (ks10825n) [inline client/src/components/Overview/Overview.jsx:50]: Kye and I were talking earlier in the week, and we were saying at this point we could probably start throwing the errors instead console logging them. We still have a week to go through our tech-debt but at the very least we should put up a ticket in trello so we don't forget about it.

File: webpack.config.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const config = {
     }
   },
   plugins: [
-    new MomentLocalesPlugins()
```
Comment (ks10825n) [inline webpack.config.js:45]: Sorry! This is my fault. It didn't give me an error until I restarted my webpack. 
Here was the docs I was looking at [https://momentjs.com/docs/#/use-it/webpack/](https://momentjs.com/docs/#/use-it/webpack/).
Now that I'm reading it again, we should have imported something on top of the config but at this point I don't think it really matters. It's more of a making our bundle less heavy. I'm okay in just removing the whole plugin configuration from line 44-46!

**Review by ks10825n — COMMENTED** [review]
Looks good! Left comments about console log and my error in the webpack config

**Comment by leonardodeutsch** [comment]
@kyelindholm @ks10825n 👋

- Implemented style selector

File: client/src/components/Overview/StyleSelector.jsx [hunk 3]
```diff
@@ -8,10 +8,24 @@ class StyleSelector extends React.Component {
     }
   }
 
+  handleSelectStyle() {
```
Comment (ks10825n) [inline client/src/components/Overview/StyleSelector.jsx:11]: Do you still need this function if you're passing in handleChangeStyle from the Overview component?

Comment (leonardodeutsch) [reply client/src/components/Overview/StyleSelector.jsx]: removed it, thanks

Comment (leonardodeutsch) [reply client/src/components/Overview/StyleSelector.jsx]: removed the ratings component from app.jsx and put it inside the overview component

**Comment by leonardodeutsch** [comment]
fixed conflicts, waiting for approval to merge @kyelindholm @ks10825n

**Comment by leonardodeutsch** [comment]
@ks10825n 

- I added the ratings component to overview component with your new implementation of ratingAverage.js and Ratings.jsx

Note: for some reason the return value of ratingAverage.js was showing as a string on my end, so you have to parse it before you pass it to Ratings.jsx

- Changed error console logs to throw errors

**Review by ks10825n — APPROVED** [review]
Looks good! Not sure why the ratingAverage function is returning a string since we're rounding with Math.round before returning it, but good to know we have to parse it.

**Pull request merged**